axlearn/common/metrics_classification.py

```diff
@@ -172,6 +177,9 @@ def binary_clf_curve(
     fps = jnp.where(y_pred_diff, fps, jnp.iinfo(jnp.int32).max)
     fps = jax.lax.cummin(fps, reverse=True)
 
+    # Masked entries at the end of thresholds are set to jnp.finfo(jnp.float32).max.
+    # Reset them to the rightmost unmasked threshold value.
+    thresholds = jnp.where(weight, thresholds, thresholds[jnp.argsort(weight)[-1]])
```

Comment:
Thanks, as discussed offline maybe we should remove this to simplify, since the masked values should not be used anyway, and I'm not sure that `jnp.argsort(weight)[-1]` gives the rightmost unmasked value.

Reply:
Done